Source/Fuse.Common/Internal/MiniList.uno

```diff
@@ -11,16 +20,26 @@ namespace Fuse.Internal
 	struct MiniList<T> : IList<T> where T : class
 	{
 		object _list;
+		ObjectList<T> AsList { get { return (ObjectList<T>)_list; } }
+		
+		MiniListMode _mode = MiniListMode.Empty;
```

Comment:
The immediate reason was to fix the 1-item mode, which wasn't working correctly and wasn't clear to fix without the enum. (It could support null again now, but I haven't lifted the restriction yet)

The second reason is for performance. The dynamic type checking is costing us time. This is showing up in loops of the form `for (int i=0; i < list.Count; ++i) { foo(list[i]); }` as each iteraction involved several dynamic type casts and comparison. These lists and loops are occuring in critical code paths, so the chioce of speed outweighs the increase in size (which as a whole isn't much for the entire app: several thousand lists are created in total).